packages/react-select/src/components/LiveRegion.tsx

```diff
@@ -112,7 +112,7 @@ const LiveRegion = <
     const isSelected = !!(
       focusedOption &&
       selectValue &&
-      selectValue.includes(focusedOption)
+      selectValue.some(option => option.value === focusedOption.value)
```

Comment:
Should use `getOptionValue` from `selectProps` to ensure custom option format compatibility.